test/python/circuit/test_commutation_checker.py

```diff
@@ -483,6 +485,18 @@ def test_nonfloat_param(self):
         self.assertTrue(scc.commute(pauli_gate, [0, 1], [], rx_gate_theta, [0], []))
         self.assertTrue(scc.commute(rx_gate_theta, [0], [], pauli_gate, [0, 1], []))
 
+    def test_2q_pauli_rot_with_non_cached(self):
+        """Test the 2q-Pauli rotations with a gate that is not cached."""
+        x_equiv = UGate(np.pi, -np.pi / 2, np.pi / 2)
+        self.assertTrue(scc.commute(x_equiv, [0], [], RXXGate(np.pi / 2), [0, 1], []))
+        self.assertTrue(scc.commute(x_equiv, [1], [], RXXGate(np.pi / 2), [0, 1], []))
+
+        something_else = RGate(1, 2)
+        self.assertTrue(scc.commute(x_equiv, [1], [], RXXGate(np.pi / 2), [0, 1], []))
+
+        self.assertFalse(scc.commute(something_else, [1], [], RYYGate(np.pi), [1, 0], []))
+        self.assertFalse(scc.commute(something_else, [1], [], RYYGate(np.pi), [1, 0], []))
```

Comment:
is this duplication intentional?
```suggestion
        self.assertFalse(scc.commute(something_else, [1], [], RYYGate(np.pi), [1, 0], []))
```

Reply:
Good catch -- one of them should be `[0]` instead of `[1]` 😅

Reply:
Fixed in 98e6d3069!